<launch>
	<group ns="ribbon_bridge_measurement" >
		<param name="sub_img_topic_name" type="str" value="/usb_cam/image_raw" />
		<param name="show_result_image" type="bool" value="True" />
		<param name="pub_result_image" type="bool" value="True" />
		<param name="ribbon_bridge_area_threshold" type="double" value="13000" />

		<node pkg="ribbon_bridge_measurement" type="measurement_for_multi_tracker_node" name="measurement_for_multi_tracker_node" output="screen" />

		<!--node pkg="ribbon_bridge_measurement" type="update_regions_of_interest_node" name="update_regions_of_interest_node" /-->

	</group>
</launch>
